data.json

```diff
@@ -0,0 +1 @@
+{"json": {"glm": {"function_handler": 
```

Comment:
What is this?

Reply:
I didn't mean to push my commit, merging two branches must have caused to push the code. I deleted that file